DfE.Hexagonal.Common/Validation/ValidationResult.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DfE.Hexagonal.Common.Validation
+{
+    /// <summary>
+    /// contains a validation result and a dictionary of errors, if applicable
+    /// </summary>
+    public class ValidationResult
+    {
+        /// <summary>
+        /// is the thing valid?
+        /// </summary>
+        public bool IsValid { get; }
```

Comment:
Could `IsValid` become a computed field/property?

Perhaps something like this:
```csharp
public bool IsValid => Errors.IsEmpty;
```

If there is a distinction to be made re: errors and warnings, perhaps we could add the warnings also?